src/env.ts

```diff
@@ -141,10 +141,25 @@ function getPropertyFromPath(obj: any, path: string[]) {
   return obj;
 }
 
+/*
+  ~check that plugins exists, if it does not then return false
+  ~decide if you have a normal array object or a enhanced object
```

Comment:
I left these comments here to make reviewing easier however I will remove them before merging.

Reply:
In the future, no need to put these comments in for reviewing 🙂 ideally the code should be clear enough to understand on its own, as yours is! If you do want to provide more context for reviewing I would suggest putting it in the PR description instead of in the code itself.